### DISCUSSION TITLE
Getting Periodic Player Stats (Free Agents)

### DISCUSSION THREAD

**Discussion by beffypuffs** [cwendt94/espn-api#440] [Q&A]
Sorry if this has been answered already, tried to find other discussions but couldn't find what I'm looking for. I'm just looking for a way to grab periodic stats for free agents (i.e. 30-day, 15-day averages). These are tracked by ESPN so I figure there is an easy way to accomplish this.

**Comment by cwendt94** [comment]
What sport is this for? There is a function in the league interface
```python
# position inputs: 'PG', 'SG', 'SF', 'PF', 'C', 'G', 'F'
def free_agents(week: int = None, size: int = 50, position: str = None, position_id: int=None) -> List[Player]:
```
this will return the [player class](https://github.com/cwendt94/espn-api/wiki/Player-Class-Basketball) that has the player stats.
